service/chat/auth.py

```diff
@@ -101,7 +101,7 @@ def main():
                 allow()
             else:
                 deny()
-        except struct.error as e:
+        except:
```

Comment:
Entering the chat container then running `mongooseimctl debug`, then within that shell, `recon:proc_count(message_queue_len, 10).`, suggests that the queue is monotonically increasing. I'm guessing that MongooseIM is getting ghosted by the `auth.py` script in some cases.